.cirrus.yml

```diff
@@ -707,7 +707,25 @@ rootless_integration_test_task:
     # Docs: ./contrib/cirrus/CIModes.md
     only_if: *not_tag_branch_build_docs_machine
     depends_on: *build
-    matrix: *platform_axis
+    # FIXME: Fails on Debian rootless ref: #22625
+    # matrix: *platform_axis
+    # FIXME: Fails on Debian rootless ref: #22625
```

Comment:
WTF????? Is this the pasta thing? Do we have a timeline for resolving this?

Reply:
I mean, IIRC the bug had something to do with the move to `/var/tmp`, did anyone consider setting an explicit `TMPDIR=/tmp` in the debian CI?

Reply:
Yes my tmpfs PR will fix it by using /tmp/... Otherwise I need to dig into apparmor